bindings/python/tools/python/ostk/astrodynamics/display.py

```diff
@@ -121,36 +129,27 @@ def __init__(
         interval: RealInterval,
         trajectory_step: Duration,
         access_step: Duration,
-        ground_station_lla: LLA,
-        color: str,
+        ground_station_lla: LLA | None = None,
```

Comment:
how can you make an access plot without supplying a ground station for which to generate accesses to 👀 ?